QAMastersHybrid2020.iml

```diff
@@ -1,2 +1,57 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we should not upload .iml file and .idea folder

Reply:
Ok, by mistake I uploaded